ztp/source-crs/du-profile-kube-compare/optional/metadata.yaml

```diff
@@ -0,0 +1,33 @@
+Parts:
```

Comment:
Asking out of curiosity why did you decide to create 2 separate references (2 metadata.yaml files) ?

Reply:
No reason...just wanted to have smaller files. is this a recommend pattern in kube-compare to have everything in one file?

Reply:
Combined to one metadata